gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/authentication/provider/generator/JWTGenerator.java

```diff
@@ -162,4 +158,8 @@ public void afterPropertiesSet() {
             LOGGER.warn("");
         }
     }
+
+    private String signingKeySecret() {
+        return configuration.getProperty("jwt.secret", "s3cR3t4grAv1t3310AMS1g1ingDftK3y");
```

Comment:
**suggestion:** these value `s3cr3t...` is used in multiple places, could it be useful to have it as a Constant somewhere to make the definition only once?

Reply:
Done, moved to the ConstantKeys